app/views/works/_all_works.html.erb

```diff
@@ -0,0 +1,18 @@
+<table class="table">    
+  <tr>
+    <th>Votes</th>
+    <th>Title</th>
+    <th>Created By</th>
+    <th>Published</th>
+    <th>Upvote</th>
+  </tr> 
+  <% Work.sorted_works(category).each do |work| %>
+    <tr>
+      <td><%= work.votes.count %></td>
```

Comment:
I'm not a huge fan of calling a model method directly from the view like this - would it be possible to "tee up" this data in the controller?